### PR TITLE
refactor: Get the simpleExchange basically working

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
orderbook sometimes has no buy or no sell.

The brandRegKeys are coming out as 'Asset', and 'Price'. deal with
  that rather than fixing it.

### CODE REVIEW

File: api/src/handler.js [hunk 1]
```diff
@@ -1,45 +1,36 @@
...
   E(bookNotifierP).getUpdateSince().then(orders => {
-    debugger;
     handleBookOrderUpdate(orders);
 
   });
   let recentOrders;
 
   const jsonAmount = ({ extent, brand }) =>
-    ({ extent, brandRegKey: brandToBrandRegKey.get(brand) });
+    ({ extent, brandRegKey: brandToKeyword.get(brand) });
```
Comment (michaelfig) [inline api/src/handler.js:21]: Please change this field into `keyword` instead of `brandRegKey`.

File: ui/src/components/OrderBook.jsx [hunk 2]
```diff
@@ -97,7 +98,7 @@ export default function OrderBook({ title, orderbook, orderBookKind}) {
               {orders
                 .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
                 .filter(({ Asset: { brandRegKey: AssetBrandRegKey }, Price: { brandRegKey: PriceBrandRegKey } }) =>
```
Comment (michaelfig) [inline ui/src/components/OrderBook.jsx:100]: Here too, `brandRegKey` becomes `keyword`, `AssetKeyword` and `PriceKeyword`.

**Review by michaelfig — CHANGES_REQUESTED** [review]
LGTM, modulo some confusing variable names.

**Review by katelynsills — APPROVED** [review]
I'm still getting some errors, but this seems like a great improvement. I think we should merge and then fix the errors separately.

When I made orders and they get matched, they don't show up in the order history. Also, somehow the inclusion of one more order closed a bunch of them. 

Screen Shot 2020-04-21 at 5 20 25 PM

Screen Shot 2020-04-21 at 5 21 09 PM


Also, I got this error in the `agoric start` terminal:
Screen Shot 2020-04-21 at 5 22 22 PM

**Comment by Chris-Hibbert** [comment]
These changes have been incorporated into #9 directly. There were enough changes on different branches that this seemed the simplest way to get a reviewable PR.

**Comment by Chris-Hibbert** [comment]
closed by #9

**Pull request closed without merging**